steps_mt_bench.md

```diff
@@ -0,0 +1,37 @@
+# MT-Bench Broken Down in Eval Scripts (not PR Bench)
```

Comment:
@danmcp will this file eventually get removed? Maybe we evolve it into the README for the repo?

Reply:
I merged with the README for now.  Still work to do obviously.